### PR TITLE
Implement Grafana viewer

### PR DESCRIPTION
**Pull request by reidmv**
So that metrics from multiple servers can be uploaded and viewed sanely in a local (Docker) Grafana/Graphite instance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,27 +62,39 @@ Examples:
...
 
 Examples:
 
-    ./export_to_graphite.rb ~/Downloads/wells_fargo/puppet_server/*.json | nc localhost 2003
+    ./json2graphite.rb ~/Downloads/wells_fargo/puppet_server/*.json | nc localhost 2003
 
 The simple example can be used for small numbers of files. When more files exist than can be referenced as arguments, use xargs.
 
-    find ~/Downloads/wells_fargo/puppet_server -name '*.json' | xargs ./export_to_graphite.rb | nc localhost 2003
+    find ~/Downloads/wells_fargo/puppet_server -name '*.json' | xargs ./json2graphite.rb | nc localhost 2003
```
Comment (acidprime) [inline README.md]: @reidmv *cough* `$bank` might be better here

Comment (reidmv) [inline README.md]: @acidprime what's `$bank`?

Comment (acidprime) [inline README.md]: heh, I just meant it would be good not to have custom names on a public repo.

Comment (reidmv) [inline README.md]: 👍

**Pull request merged**